lighthouse-core/scripts/legacy-javascript/run.js

```diff
@@ -177,7 +62,8 @@ async function createVariant(options) {
     fs.writeFileSync(`${dir}/.babelrc`, JSON.stringify(babelrc || {}, null, 2));
     // Not used in this script, but useful for running Lighthouse manually.
     // Just need to start a web server first.
-    fs.writeFileSync(`${dir}/index.html`, `<title>${name}</title><script src=main.bundle.min.js>`);
```

Comment:
a nonclosing script element is subtly invalid. Chrome/FF devtools still shows a JS network request, but no reason why the code isn't running. this bug didn't matter because the html file is only for loading the variant page manually (it isnt used in any of the following steps)